src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Tab from "./Tab";
+import StyledTabs from "./Tabs.styled";
+
+const Tabs = (props) => {
+	const initializeState = (tabs) => {
```

Comment:
Raczej w state przechowywałbym tylko informacje jaki element jest aktywny czyli coś takiego: `initActiveTab = 'home'`. (state to tylko ciąg znaków, który jest ustawiany z `eventKey`.